### PR TITLE
DBZ-705: Postgres alpine images

### PR DESCRIPTION
**Pull request by csdev**
Build Alpine Linux versions of the postgres images.
These images are much smaller than the default Debian-based images.

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Thanks, @csdev! Could you file a [JIRA issue](https://issues.jboss.org/browse/DBZ) and reference its number in the commit message?

File: postgres/10.0-alpine/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+The [Postgres](https://www.postgresql.org) relational database management system has a feature called [logical decoding](https://www.postgresql.org/docs/9.4/static/logicaldecoding-explanation.html) that allows clients to extract all persistent changes to a database's tables into a coherent, easy to understand format which can be interpreted without detailed knowledge of the database's internal state. An output plugin transform the data from the write-ahead log's internal representation into the format the consumer of a replication slot desires.
+
+This image is based upon [`postgres:10.0-alpine`](https://hub.docker.com/_/postgres/) and adds a Debezium-specific [logical decoding output plugin that uses Protobuf](https://github.com/debezium/postgres-decoderbufs), enabling the [Debezium PostgreSQL Connector](http://debezium.io/docs/connectors/postgresql/) to capture changes committed to the database and record the data change events in Kafka topics. The image also includes [PostGIS](http://www.postgis.net) spatial database extender used to provide geospatial queries, so that changes to geometric data can also be captured by Debezium.
```
Comment (gunnarmorling) [inline postgres/10.0-alpine/README.md]: I suppose it's copied from the existing one, but it'd be good to clarify that there are two LD plug-ins actually by now: the ProtoBuf-based one and another one based on wal2json.

**Comment by gunnarmorling** [comment]
@jpechane I think it's a good addition. Once it's merged we also should set up jobs for testing on CI.

**Comment by gunnarmorling** [comment]
Hey @csdev, just to let you know that this PR is not forgotten. We're in the process of releasing 0.8.0.Beta1 and this requires quite some changes to the release process (as we have a new separate repo now with incubating connectors). So we'll get back to this one once the dust has settled. Sorry for the delay, it's definitely a good change and I'd like to publish these images eventually.

**Comment by jpechane** [comment]
@csdev, Applied, thanks, images should be now built automatically

**Comment by jpechane** [comment]
@csdev I am sorry but I've to remove the PR, I got this error later on on of the systems
```
WARNING: This apk-tools is OLD! Some packages might not function properly.
(1/1) Installing json-c (0.13.1-r0)
OK: 28 MiB in 34 packages
fetch http://dl-cdn.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.7/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.7/community/x86_64/APKINDEX.tar.gz
WARNING: This apk-tools is OLD! Some packages might not function properly.
ERROR: unsatisfiable constraints:
  so:libcrypto.so.43 (missing):
    required by:
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
                 gdal-2.3.0-r0[so:libcrypto.so.43]
The command '/bin/sh -c if [ "$USE_POSTGIS" != "false" ]; then         apk add --no-cache --repository 'http://dl-cdn.alpinelinux.org/alpine/edge/main' json-c         && apk add --no-cache --repository 'http://dl-cdn.alpinelinux.org/alpine/edge/testing'             postgis-dev proj4-dev;     fi' returned a non-zero code: 4
```

**Comment by csdev** [comment]
@jpechane Some of the packages required for postgis support are only available on alpine edge, and it looks like that branch is too unreliable right now. I'll try to find a way to make this build more reproducible, but if that doesn't work, we may want to wait for the next stable alpine release.

**Pull request merged**